catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/BackupCommand.java

```diff
@@ -116,4 +172,140 @@ private String getBackupUrl(String coreName) {
         }
         return backupUrl + "/" + coreName + "/replication";
     }
+
+    private boolean isSystemConfiguredWithSolrCloud() {
+        String solrClientType = System.getProperty(SOLR_CLIENT_PROP);
+        LOGGER.debug("solr client type: {}", solrClientType);
+        if (solrClientType != null) {
+            return StringUtils.equals(solrClientType, CLOUD_SOLR_CLIENT_TYPE);
+        } else {
+            printErrorMessage(String.format(
+                    "Could not determine Solr Client Type. Please verify that the system property %s is configured in %s.",
+                    SOLR_CLIENT_PROP, Paths.get(System.getProperty("karaf.home"), "etc", "system.properties")));
```

Comment:
I thought we were generally trying to abstract `karaf.home` and use `ddf.home` is that no longer the case?